content/websockets/adapter.md

```diff
@@ -30,21 +30,21 @@ The WebSockets module is platform-agnostic, hence, you can bring your own librar
 The [socket.io](https://github.com/socketio/socket.io) package is wrapped in an `IoAdapter` class. What if you would like to enhance the basic functionality of the adapter? For instance, your technical requirements require a capability to broadcast events across multiple load-balanced instances of your web service. For this, you can extend `IoAdapter` and override a single method which responsibility is to instantiate new socket.io servers. But first of all, let's install the required package.
 
 ```bash
-$ npm i --save socket.io-redis
+$ npm i --save @socket.io/redis-adapter
 ```
 
 Once the package is installed, we can create a `RedisIoAdapter` class.
 
 ```typescript
 import { IoAdapter } from '@nestjs/platform-socket.io';
-import { RedisClient } from 'redis';
-import { ServerOptions } from 'socket.io';
-import { createAdapter } from 'socket.io-redis';
+import { Server, ServerOptions } from 'socket.io';
```

Comment:
as you're using `socket.io`, you need to install it as well, right? 

![image](https://user-images.githubusercontent.com/13461315/153673246-2743e13a-b23f-44a7-bb56-3db7ac160735.png)

![image](https://user-images.githubusercontent.com/13461315/153673371-c46f1997-2f57-4a30-a4c6-2d580f222703.png)

Reply:
Sounds good